### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by AlJohri** [tom-dalton-fanduel/fdjangodog#8]
I think it would be helpful to document:

- this does not need datadog APM
- what is datadog APM
- if you use datadog APM, use [dd-trace-py](http://pypi.datadoghq.com/trace/docs/) instead
- this does not use the datadog API (and thus does not need a datadog api key)
- why not use the datadog api / what is the difference between using the api vs. statsd
- if you need to use the datadog API, you can try using [dj-datadog](https://github.com/krmboya/dj-datadog)

Some of this might be out of scope, but getting started with integrating datadog I had a lot of trouble figuring out what to use and why.

**Comment by tom-dalton-fanduel** [comment]
I agree that this package is quite different than https://github.com/conorbranagan/django-datadog in terms of using stasd over the api. I'm happy to accept PRs to improve the documentation in general, and specifically when/why to use this package vs alternatives.

**Comment by ogdirty** [comment]
> I think it would be helpful to document:
> 
> * this does not need datadog APM
> * what is datadog APM
> * if you use datadog APM, use [dd-trace-py](http://pypi.datadoghq.com/trace/docs/) instead
> * this does not use the datadog API (and thus does not need a datadog api key)
> * why not use the datadog api / what is the difference between using the api vs. statsd
> * if you need to use the datadog API, you can try using [dj-datadog](https://github.com/krmboya/dj-datadog)
> 
> Some of this might be out of scope, but getting started with integrating datadog I had a lot of trouble figuring out what to use and why.

Tryusing Dat11/ noo".
